### PR TITLE
Feature/ie 8

### PR DESCRIPTION
**Pull request by dspasojevic**

### CODE REVIEW

File: src/scripts/deactivate-user.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  oktaManageClient,
+  OktaConfiguration,
+  user,
+  User,
+} from './services/user-service';
+
+const deactivateUser = async (
+  oktaConfiguration: OktaConfiguration,
+  userId: string
+): Promise<User> => {
```
Comment (ByronDobner) [inline src/scripts/deactivate-user.ts:14]: Make this array readonly

File: src/scripts/deactivate-user.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      // eslint-disable-next-line functional/no-try-statement
+      try {
+        const user = await deactivateUser(
+          {
+            ...args,
+          },
+          args.userId
+        );
+        // eslint-disable-next-line functional/no-expression-statement
+        console.info(user);
```
Comment (ByronDobner) [inline src/scripts/deactivate-user.ts:63]: nit: output a success message?

Comment (dspasojevic) [reply src/scripts/deactivate-user.ts:63]: Good idea. I've got that change queued up as part of the next PR.

File: src/scripts/services/user-service.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+   */
+  readonly name: string;
+  /**
+   * User status as a string.
+   */
+  readonly status: string;
+};
+
+// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+export const user = (oktaUser: okta.User) => ({
```
Comment (ByronDobner) [inline src/scripts/services/user-service.ts:29]: Is there a better funtion name than `user` that describes this function? Something like `oktaUserToUser` or `simplifiyOktaUser`

Comment (dspasojevic) [reply src/scripts/services/user-service.ts:29]: Yep, good question. 

Where possible, we tend to reserve the use of verbs for functions that have side effects.

What about `oktaUserAsUser` ?

**Review by ByronDobner — APPROVED** [review]

**Pull request merged**